src/Serialize.h

```diff
@@ -239,7 +239,7 @@ namespace drafter {
         }
 
         template<typename Collection, typename Functor, typename Arg1, typename Arg2>
-        R operator()(const Collection& collection, Functor &wrapper, Arg1 arg1, Arg2 arg2) const {
+        R operator()(const Collection& collection, Functor &wrapper, Arg1 arg1, Arg2& arg2) const {
```

Comment:
And reason why must be arg2 by reference?
There is option use `std::ref()` if reference is required,